pull_request_template.md

```diff
@@ -3,9 +3,13 @@ Write a detailed description of your PR here.
 Please follow the guidelines for PRs contained [here](docs/source/contributing.md). Checklist:
```

Comment:
Can we move the contributing document to a folder that will be found by GitHub to comply with [community standards](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines)?